console/src/main/scripts/plugins/metrics/ts/services/alertsManager.ts

```diff
@@ -363,12 +363,12 @@ module HawkularMetrics {
                   }
                 }
 
-                if ( evalItem.rate ) {
+                if ( undefined != evalItem.rate ) {
```

Comment:
I'm pretty sure that *if( !evalItem )* will get the falsy condition of undefined